File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/commands/SelectRowCommand_View.cpp

#include "SelectRowCommand_View.h"
#include "SelectRowCommand_View.h"
#include <vector>
#include <memory>
#include "SelectRowCommand_ViewMyListViewElementsRow.h"
#include "SelectRowCommand_ViewMyStringRowHandleListViewElementsRow.h"
#include "SelectRowCommand_ViewMyTableViewElementsRow.h"
#include "SelectRowCommand_ViewMyStringRowHandleTableViewElementsRow.h"
#include "SelectRowCommand_ViewMyTreeViewElementsRow.h"
#include "SelectRowCommand_ViewMyStringRowHandleTreeViewElementsRow.h"
#include "SelectRowCommand_ViewMyIndexBasedMultiSelectRowListViewElementsRow.h"
#include "SelectRowCommand_ViewMyStringHandleMultiSelectRowListViewElementsRow.h"
#include <string>

namespace commands
{
  std::vector<std::shared_ptr<SelectRowCommand_ViewMyListViewElementsRow>> SelectRowCommand_View::getMyListViewElementsListRows( ) 
  {
    return this->MyListViewElementsListRows;
  }
  int SelectRowCommand_View::getMyListViewElementsListSelectedRow( ) 
  {
    return this->MyListViewElementsListSelectedRow;
  }
  std::vector<std::shared_ptr<SelectRowCommand_ViewMyStringRowHandleListViewElementsRow>> SelectRowCommand_View::getMyStringRowHandleListViewElementsListRows( ) 
  {
    return this->MyStringRowHandleListViewElementsListRows;
  }
  std::string SelectRowCommand_View::getMyStringRowHandleListViewElementsListSelectedRow( ) 
  {
    return this->MyStringRowHandleListViewElementsListSelectedRow;
  }
  std::vector<std::shared_ptr<SelectRowCommand_ViewMyTableViewElementsRow>> SelectRowCommand_View::getMyTableViewElementsTableRows( ) 
  {
    return this->MyTableViewElementsTableRows;
  }
  int SelectRowCommand_View::getMyTableViewElementsTableSelectedRow( ) 
  {
    return this->MyTableViewElementsTableSelectedRow;
  }
  std::vector<std::shared_ptr<SelectRowCommand_ViewMyStringRowHandleTableViewElementsRow>> SelectRowCommand_View::getMyStringRowHandleTableViewElementsTableRows( ) 
  {
    return this->MyStringRowHandleTableViewElementsTableRows;
  }
  std::string SelectRowCommand_View::getMyStringRowHandleTableViewElementsTableSelectedRow( ) 
  {
    return this->MyStringRowHandleTableViewElementsTableSelectedRow;
  }
  std::vector<std::shared_ptr<SelectRowCommand_ViewMyTreeViewElementsRow>> SelectRowCommand_View::getMyTreeViewElementsTreeRows( ) 
  {
    return this->MyTreeViewElementsTreeRows;
  }
  int SelectRowCommand_View::getMyTreeViewElementsTreeSelectedRow( ) 
  {
    return this->MyTreeViewElementsTreeSelectedRow;
  }
  std::vector<std::shared_ptr<SelectRowCommand_ViewMyStringRowHandleTreeViewElementsRow>> SelectRowCommand_View::getMyStringRowHandleTreeViewElementsTreeRows( ) 
  {
    return this->MyStringRowHandleTreeViewElementsTreeRows;
  }
  std::string SelectRowCommand_View::getMyStringRowHandleTreeViewElementsTreeSelectedRow( ) 
  {
    return this->MyStringRowHandleTreeViewElementsTreeSelectedRow;
  }
  std::vector<std::shared_ptr<SelectRowCommand_ViewMyIndexBasedMultiSelectRowListViewElementsRow>> SelectRowCommand_View::getMyIndexBasedMultiSelectRowListViewElementsListRows( ) 
  {
    return this->MyIndexBasedMultiSelectRowListViewElementsListRows;
  }
  std::vector<int> SelectRowCommand_View::getMyIndexBasedMultiSelectRowListViewElementsListSelectedRows( ) 
  {
    return this->MyIndexBasedMultiSelectRowListViewElementsListSelectedRows;
  }
  std::vector<std::shared_ptr<SelectRowCommand_ViewMyStringHandleMultiSelectRowListViewElementsRow>> SelectRowCommand_View::getMyStringHandleMultiSelectRowListViewElementsListRows( ) 
  {
    return this->MyStringHandleMultiSelectRowListViewElementsListRows;
  }
  std::vector<std::string> SelectRowCommand_View::getMyStringHandleMultiSelectRowListViewElementsListSelectedRows( ) 
  {
    return this->MyStringHandleMultiSelectRowListViewElementsListSelectedRows;
  }
  void SelectRowCommand_View::myListViewElementsRowSelected(int rowIndex ) 
  {
    
  }
  void SelectRowCommand_View::myStringRowHandleListViewElementsRowSelected(std::string rowHandle ) 
  {
    
  }
  void SelectRowCommand_View::myTableViewElementsRowSelected(int rowIndex ) 
  {
    
  }
  void SelectRowCommand_View::myStringRowHandleTableViewElementsRowSelected(std::string rowHandle ) 
  {
    
  }
  void SelectRowCommand_View::myTreeViewElementsRowSelected(int rowIndex ) 
  {
    
  }
  void SelectRowCommand_View::myStringRowHandleTreeViewElementsRowSelected(std::string rowHandle ) 
  {
    
  }
  void SelectRowCommand_View::myIndexBasedMultiSelectRowListViewElementsRowSelected(std::vector<int> rowIndices ) 
  {
    
  }
  void SelectRowCommand_View::myStringHandleMultiSelectRowListViewElementsRowSelected(std::vector<std::string> rowHandles ) 
  {
    
  }
}

File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/commands/SelectRowCommand_View.cs
/// <filename>
///     SelectRowCommand_View.cs
/// </filename>
using System.Collections.Generic;
using commands;

namespace commands
{
    internal class SelectRowCommand_View
  {
        private System.Collections.Generic.List<commands.SelectRowCommand_ViewMyListViewElementsRow> MyListViewElementsListRows = new System.Collections.Generic.List<commands.SelectRowCommand_ViewMyListViewElementsRow>();
     
        public     virtual System.Collections.Generic.List<commands.SelectRowCommand_ViewMyListViewElementsRow> getMyListViewElementsListRows()
    {
      return this.MyListViewElementsListRows;
    }
     
        private int MyListViewElementsListSelectedRow;
     
        public     virtual int getMyListViewElementsListSelectedRow()
    {
      return this.MyListViewElementsListSelectedRow;
    }
     
        private System.Collections.Generic.List<commands.SelectRowCommand_ViewMyStringRowHandleListViewElementsRow> MyStringRowHandleListViewElementsListRows = new System.Collections.Generic.List<commands.SelectRowCommand_ViewMyStringRowHandleListViewElementsRow>();
     
        public     virtual System.Collections.Generic.List<commands.SelectRowCommand_ViewMyStringRowHandleListViewElementsRow> getMyStringRowHandleListViewElementsListRows()
    {
      return this.MyStringRowHandleListViewElementsListRows;
    }
     
        private string MyStringRowHandleListViewElementsListSelectedRow;
     
        public     virtual string getMyStringRowHandleListViewElementsListSelectedRow()
    {
      return this.MyStringRowHandleListViewElementsListSelectedRow;
    }
     
        private System.Collections.Generic.List<commands.SelectRowCommand_ViewMyTableViewElementsRow> MyTableViewElementsTableRows = new System.Collections.Generic.List<commands.SelectRowCommand_ViewMyTableViewElementsRow>();
     
        public     virtual System.Collections.Generic.List<commands.SelectRowCommand_ViewMyTableViewElementsRow> getMyTableViewElementsTableRows()
    {
      return this.MyTableViewElementsTableRows;
    }
     
        private int MyTableViewElementsTableSelectedRow;
     
        public     virtual int getMyTableViewElementsTableSelectedRow()
    {
      return this.MyTableViewElementsTableSelectedRow;
    }
     
        private System.Collections.Generic.List<commands.SelectRowCommand_ViewMyStringRowHandleTableViewElementsRow> MyStringRowHandleTableViewElementsTableRows = new System.Collections.Generic.List<commands.SelectRowCommand_ViewMyStringRowHandleTableViewElementsRow>();
     
        public     virtual System.Collections.Generic.List<commands.SelectRowCommand_ViewMyStringRowHandleTableViewElementsRow> getMyStringRowHandleTableViewElementsTableRows()
    {
      return this.MyStringRowHandleTableViewElementsTableRows;
    }
     
        private string MyStringRowHandleTableViewElementsTableSelectedRow;
     
        public     virtual string getMyStringRowHandleTableViewElementsTableSelectedRow()
    {
      return this.MyStringRowHandleTableViewElementsTableSelectedRow;
    }
     
        private System.Collections.Generic.List<commands.SelectRowCommand_ViewMyTreeViewElementsRow> MyTreeViewElementsTreeRows = new System.Collections.Generic.List<commands.SelectRowCommand_ViewMyTreeViewElementsRow>();
     
        public     virtual System.Collections.Generic.List<commands.SelectRowCommand_ViewMyTreeViewElementsRow> getMyTreeViewElementsTreeRows()
    {
      return this.MyTreeViewElementsTreeRows;
    }
     
        private int MyTreeViewElementsTreeSelectedRow;
     
        public     virtual int getMyTreeViewElementsTreeSelectedRow()
    {
      return this.MyTreeViewElementsTreeSelectedRow;
    }
     
        private System.Collections.Generic.List<commands.SelectRowCommand_ViewMyStringRowHandleTreeViewElementsRow> MyStringRowHandleTreeViewElementsTreeRows = new System.Collections.Generic.List<commands.SelectRowCommand_ViewMyStringRowHandleTreeViewElementsRow>();
     
        public     virtual System.Collections.Generic.List<commands.SelectRowCommand_ViewMyStringRowHandleTreeViewElementsRow> getMyStringRowHandleTreeViewElementsTreeRows()
    {
      return this.MyStringRowHandleTreeViewElementsTreeRows;
    }
     
        private string MyStringRowHandleTreeViewElementsTreeSelectedRow;
     
        public     virtual string getMyStringRowHandleTreeViewElementsTreeSelectedRow()
    {
      return this.MyStringRowHandleTreeViewElementsTreeSelectedRow;
    }
     
        private System.Collections.Generic.List<commands.SelectRowCommand_ViewMyIndexBasedMultiSelectRowListViewElementsRow> MyIndexBasedMultiSelectRowListViewElementsListRows = new System.Collections.Generic.List<commands.SelectRowCommand_ViewMyIndexBasedMultiSelectRowListViewElementsRow>();
     
        public     virtual System.Collections.Generic.List<commands.SelectRowCommand_ViewMyIndexBasedMultiSelectRowListViewElementsRow> getMyIndexBasedMultiSelectRowListViewElementsListRows()
    {
      return this.MyIndexBasedMultiSelectRowListViewElementsListRows;
    }
     
        private System.Collections.Generic.List<int> MyIndexBasedMultiSelectRowListViewElementsListSelectedRows = new System.Collections.Generic.List<int>();
     
        public     virtual System.Collections.Generic.List<int> getMyIndexBasedMultiSelectRowListViewElementsListSelectedRows()
    {
      return this.MyIndexBasedMultiSelectRowListViewElementsListSelectedRows;
    }
     
        private System.Collections.Generic.List<commands.SelectRowCommand_ViewMyStringHandleMultiSelectRowListViewElementsRow> MyStringHandleMultiSelectRowListViewElementsListRows = new System.Collections.Generic.List<commands.SelectRowCommand_ViewMyStringHandleMultiSelectRowListViewElementsRow>();
     
        public     virtual System.Collections.Generic.List<commands.SelectRowCommand_ViewMyStringHandleMultiSelectRowListViewElementsRow> getMyStringHandleMultiSelectRowListViewElementsListRows()
    {
      return this.MyStringHandleMultiSelectRowListViewElementsListRows;
    }
     
        private System.Collections.Generic.List<string> MyStringHandleMultiSelectRowListViewElementsListSelectedRows = new System.Collections.Generic.List<string>();
     
        public     virtual System.Collections.Generic.List<string> getMyStringHandleMultiSelectRowListViewElementsListSelectedRows()
    {
      return this.MyStringHandleMultiSelectRowListViewElementsListSelectedRows;
    }
     
        public     virtual void myListViewElementsRowSelected(int rowIndex)
    {
      
    }
     
        public     virtual void myStringRowHandleListViewElementsRowSelected(string rowHandle)
    {
      
    }
     
        public     virtual void myTableViewElementsRowSelected(int rowIndex)
    {
      
    }
     
        public     virtual void myStringRowHandleTableViewElementsRowSelected(string rowHandle)
    {
      
    }
     
        public     virtual void myTreeViewElementsRowSelected(int rowIndex)
    {
      
    }
     
        public     virtual void myStringRowHandleTreeViewElementsRowSelected(string rowHandle)
    {
      
    }
     
        public     virtual void myIndexBasedMultiSelectRowListViewElementsRowSelected(System.Collections.Generic.List<int> rowIndices)
    {
      
    }
     
        public     virtual void myStringHandleMultiSelectRowListViewElementsRowSelected(System.Collections.Generic.List<string> rowHandles)
    {
      
    }
  }
}

